roles/lib_openshift/src/test/unit/test_oadm_manage_node.py

```diff
@@ -176,11 +166,11 @@ def test_schedulable_false(self, mock_openshift_cmd, mock_tmpfile_copy):
 
         results = ManageNode.run_ansible(params, False)
 
-        self.assertTrue(results['changed'])
-        self.assertEqual(results['results']['nodes'][0]['name'], 'ip-172-31-49-140.ec2.internal')
-        self.assertEqual(results['results']['nodes'][0]['schedulable'], False)
+        assert results['changed']
+        assert results['results']['nodes'][0]['name'] == 'ip-172-31-49-140.ec2.internal'
+        assert results['results']['nodes'][0]['schedulable'] == False
 
-    @unittest.skipIf(six.PY3, 'py2 test only')
+    @pytest.mark.skipif(six.PY3, reason='py2 test only')
     @mock.patch('os.path.exists')
     @mock.patch('os.environ.get')
     def test_binary_lookup_fallback(self, mock_env_get, mock_path_exists):
```

Comment:
I have some fixes lined up for this, it turns the binary lookup into a common fixture for all the files and uses a parametrized test to reduce at least some of the duplication.